.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  IMAGE_TAG: ${{ github.sha }}
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build Docker images
+      run: |
+        set -e
```

Comment:
Not needed: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference